lib/vagrant_cloud/cli.rb

```diff
@@ -0,0 +1,70 @@
+require 'thor'
+
+module VagrantCloud
+  class Cli < Thor
+    package_name 'VagrantCloud'
+
+    class_option :username, alias: '-u', desc: 'Vagrant Cloud username'
+    class_option :token, alias: '-t', desc: 'Vagrant Cloud access token'
+    class_option :box, alias: '-b', desc: 'name of the box'
+    class_option :version, alias: '-v', desc: 'version within the box'
+    class_option :provider, alias: '-p', default: 'virtualbox', desc: 'the provider for the box; default: virtualbox'
+    class_option :provider_url, alias: '-pu', desc: 'URL to the file for remote hosting; either a _url or _file_path can be provided, not both'
+    class_option :provider_file_path, alias: '-pfp', desc: 'path to file to be uploaded for Atlast hosting'
```

Comment:
|`line 41`|
|---|
This can probably be removed. Or is there a reason you "return" the versions from this function?